variants/STM32L4xx/L431C(B-C)(T-U)/ldscript.ld

```diff
@@ -46,7 +46,7 @@ _Min_Stack_Size = 0x400; /* required amount of stack */
 MEMORY
 {
   RAM    (xrw)    : ORIGIN = 0x20000000,   LENGTH = LD_MAX_DATA_SIZE
-  RAM2    (xrw)    : ORIGIN = 0x10000000,   LENGTH = 16K
+  RAM2    (xrw)    : ORIGIN = 0x2001C000,   LENGTH = 16K
```

Comment:
No the value is correct. This was just a note :wink:
RAM2 is also mapped at the end of the RAM, that's all.
```suggestion
  RAM2    (xrw)    : ORIGIN = 0x10000000,   LENGTH = 16K
```

Reply:
ah okay, I thought I had seen it both ways in other variant ldscript.ld, so was not sure.

thanks